_opcodes/op116.html

```diff
@@ -10,6 +10,16 @@
 iwd2: 1
 pst: 1
 ---
-Removes invisibility from the targeted creature(s).<br/>
-This effect ignores durations attributed to it.<br/>
+Removes invisibility from the targeted creature(s).
+
+{% capture note %}
+This effect ignores durations attributed to it.
```

Comment:
This is also something not worth making a fuss about. It's a removal opcode, not an immunity. A one shot — I'm surprised people deemed it worthy to mention this at all.